docs/designers-developers/developers/tutorials/devenv/readme.md

```diff
@@ -119,3 +181,49 @@ The important part is having a WordPress site installed, and know where and how
 Alternative editors include [Sublime Text](https://www.sublimetext.com/) that is also available across platforms, though is a commercial product; or other free alternatives include [Vim](https://www.vim.org/), [Atom](https://atom.io/), and [Notepad++](https://notepad-plus-plus.org/) all support standard JavaScript style development.
 
 You can use any editor you're comfortable with, it is more a personal preference. The development setup for WordPress block editor is a common JavaScript environment and most editors have plugins and suppport. The key is having a way to open, edit, and save text files.
+
+## Uninstall - Start Over
+
+Here are a few instructions if you need to start over, or want to remove what was installed.
+
+### Local Environment
+
+* You can remove the local environment data for a specific project:
+
+```
+wp-env destroy
+```
+
+* To completely uninstall wp-env tool:
+
+```
+npm -g uninstall @wordpress/env
+```
+
+* To uninstall Docker, or Visual Studio Code use your OS method to remove packages. For example, on Windows run "Add or remove programs". You can additionally uninstall from the Docker Desktop app, click the bug icon at the top to switch to this Troubleshoot screen. Click Uninstall or remove.
+
+![Docker Troubleshoot Screenshot](https://developer.wordpress.org/files/2020/08/docker-uninstall-screen.png)
+
+
+### Uninstall Node/NVM
+
+To uninstall Node/NVM, delete the NVM directory, this is typically installed at `$HOME/.nvm`. If the `$NVM_DIR` environment variable is set you can remove using:
+
+```
+rm -rf "$NVM_DIR"
+```
+
+or, delete directly using:
+
+```
+rm -rf "$HOME/.nvm"
+```
```

Comment:
I found it confusing to have two methods to uninstall nvm.

"To uninstall Node/NVM, delete the NVM directory, this is typically installed at `$HOME/.nvm`. If the `$NVM_DIR` environment variable is set you can remove using:"

`rm -rf "$NVM_DIR"`

or, delete directly using:

`rm -rf "$HOME/.nvm"`

------

Rephrasing.
"To uninstall Node/NVM, delete the NVM directory, this is typically installed at `$HOME/.nvm`.

`rm -rf "$HOME/.nvm"`

If the above does not remove nvm then you can instead use `rm -rf "$NVM_DIR"` as the environment variables were set. 

-----------

It is important to focus on one method and have a clear if that does not work then here is an option.

Reply:
:+1: Updated